diff --git a/_build/conf.py b/_build/conf.py
index b707b25a4..376ec83f2 100644
--- a/_build/conf.py
+++ b/_build/conf.py
@@ -132,13 +132,16 @@ api_url = 'http://api.symfony.com/master/%s'
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_rtd_theme"
+html_theme = "alabaster"
 html_theme_path = ["_themes", ]
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-#html_theme_options = {}
+html_theme_options = {
+        'nosidebar': True,
+        'show_powered_by': False
+}
 
 # Add any paths that contain custom themes here, relative to this directory.
 #html_theme_path = []
